zio-http/shared/src/main/scala/zio/http/endpoint/openapi/JsonSchema.scala

```diff
@@ -318,129 +318,141 @@ object JsonSchema {
       case SegmentCodec.Combined(_, _, _) => throw new IllegalArgumentException("Combined segment is not supported.")
     }
 
-  def fromZSchemaMulti(schema: Schema[_], refType: SchemaStyle = SchemaStyle.Inline): JsonSchemas = {
+  def fromZSchemaMulti(
+    schema: Schema[_],
+    refType: SchemaStyle = SchemaStyle.Inline,
+    seen: Set[Predef.String] = Set.empty,
```

Comment:
Please don't use Predef. It is not the origin of String, but just an indirection.